_inc/client/privacy/index.jsx

```diff
@@ -98,13 +105,14 @@ class Privacy extends React.Component {
 							}
 						</p>
 						<p>
-							<ModuleToggle
+							<CompactFormToggle
 								compact
-								activated={ getOptionValue( 'jetpack_event_tracking' ) }
-								toggling={ isSavingAnyOption( 'jetpack_event_tracking' ) }
-								toggleModule={ this.togglePrivacy }>
+								checked={ ! this.props.tracking.tracks_opt_out }
+								disabled={ this.props.isFetching || this.props.isUpdating() }
```

Comment:
The `isUpdating` prop should be a boolean value, not a function. However, it's currently a function because there is a conflict with the `isUpdating` prop in `ModuleSettingsForm`. I'm taking a look at a fix.

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/9156/commits/89408c6757f31b50da13a94e8136c9ff08175b76